src/test/java/HT_tests/DeliverToTest.java

```diff
@@ -5,7 +5,9 @@
 import org.openqa.selenium.WebDriver;
```

Comment:
As I understand, this is not your functionality. I mean, this PR is for search field functionality. I guess it should be removed